src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -1918,12 +1924,21 @@ namespace NServiceBus.Performance.TimeToBeReceived
         public System.TimeSpan MaxTime { get; }
     }
 }
-namespace NServiceBus.Persistence.Legacy
-{
-    public class MsmqPersistence : NServiceBus.Persistence.PersistenceDefinition { }
-}
 namespace NServiceBus.Persistence
 {
+    public interface CompletableSynchronizedStorageSession : NServiceBus.Persistence.SynchronizedStorageSession, System.IDisposable
```

Comment:
`SynchronizedStorageSession` is the user-facing one. `CompletableSynchronizedStorageSession`  is the persistence impl. facing one. The core expects that session to have a complete method but users should never call it themselves.